library/constant_time_internal.h

```diff
@@ -74,21 +74,21 @@
 typedef size_t    mbedtls_ct_condition_t;
 typedef size_t    mbedtls_ct_uint_t;
 typedef ptrdiff_t mbedtls_ct_int_t;
-#define MBEDTLS_CT_TRUE  ((mbedtls_ct_condition_t) SIZE_MAX)
+#define MBEDTLS_CT_TRUE  ((mbedtls_ct_condition_t) mbedtls_ct_compiler_opaque(SIZE_MAX))
```

Comment:
Question: is it more efficient to make this `#define` (so the function is called each time the constant is used), or a global variable like `mbedtls_ct_zero`?

Reply:
It seems slightly worse (10 bytes) for code-size as a global variable - at least where we have asm support, so that `mbedtls_ct_compiler_opaque` compiles to nothing. I guess this would also be slower as well, because it would have to be volatile, so it would force a read from memory.

Reply:
Ok, thanks for checking!